salt/modules/dockerng.py

```diff
@@ -504,10 +504,14 @@ def __virtual__():
                 log.warning(
                     'Insufficient Docker version for dockerng (required: '
                     '{0}, installed: {1})'.format(
-                        '.'.join(docker_versioninfo),
-                        '.'.join(MIN_DOCKER)
-                    )
-                )
+                        '.'.join(map(str, MIN_DOCKER)),
+                        '.'.join(map(str, docker_versioninfo))))
+        else:
+            log.warning(
```

Comment:
@blueyed, I think so, my superficial search on 2015.5 has not found any returns out of `__virtual__` functions of this type.  There was at least one issue that discussed this, but I cannot find it.  Will you open an issue to document this?  Thanks.